modules/manage/pages/manage-security/configure-pam.adoc

```diff
@@ -88,40 +89,45 @@ If output to the above command does not confirm that `MECHANISMS` is set to `pam
 +
 [source,bash]
 ----
-$ cp /etc/pam.d/passwd /etc/pam.d/couchbase
+cp /etc/pam.d/passwd /etc/pam.d/couchbase
 ----
 
 . Create a Linux user on the current system, and give them a password.
-For example, username [.in]`don`, and password [.in]`secretpa$$`.
-(This user is the administrator who will be authenticated by PAM.)
+For example, use the username [.in]`linuxuser`
+(this user is the administrator who will be authenticated by PAM).
+Enter the following commands, to create the user and to commence definition of their password, respectively:
 +
 [source,bash]
 ----
-$ useradd don
-$ passwd secretpa$$
-Enter new UNIX password:
+useradd linuxuser
+passwd linuxuser
 ----
 +
-Enter and then verify your chosen password, as prompted.
+The `passwd` command returns the prompt `Enter new UNIX password:`.
+Duly enter and then verify your chosen password.
 
-. Access Couchbase Web Console, at `localhost:8091`, and log in.
+. Access Couchbase Web Console (if on the same node, at `localhost:8091`), and log in.
 Then, access the [.ui]*Security* tab,  on the upper, horizontal control-bar.
 This brings up the [.ui]*Security* view:
 +
 [#security_view_initial]
 image::manage-security/securityViewInitial.png[,820]
 
-. Left-click on the btn:[Add User] button, situated near the right.
+. Left-click on the *ADD USER* button, situated near the right.
 This brings up the [.ui]*Add New User* dialog.
 Select the [.ui]*External* radio-button, in the [.ui]*Authentication Domain* panel at the upper left.
-Then, enter the name of the new user you are creating, and specify a suitable role, such as [.ui]*Cluster Admin*.
-The panel appears as follows:
+Then, enter the name of the new user you are creating.
+Note that since PAM authentication has not yet taken effect, the notification `not found` now appears above the username-field.
```

Comment:
Actually this is a bug. It actually tries to search for LDAP user with provided name. It only makes sense when native LADP is enabled. We need to fix that in the UI I think. I will create a ticket for that.
Probably we can say something like that: "you might see 'not found' notification if native LDAP is enabled as well"